### PR TITLE
Update auto-merge bot

### PR DESCRIPTION
**Pull request by tmigot**

### CODE REVIEW

**Review by amontoison — APPROVED** [review]

**Comment by tmigot** [comment]
Oo... Was it expected that the probot strike again?

**Comment by amontoison** [comment]
Yes, the PR was not merged.
Can you open a new (dummy) PR to verify that `probot` works as intended now?

File: .github/auto-merge.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 rules:
 - requiredAuthorRole: MEMBER
   minApprovals:
-    MEMBER: 1
+    MEMBER: 2
```
Comment (dpo) [inline .github/auto-merge.yml:4]: Is there a way to request an approval from all reviews requested?

Comment (dpo) [reply .github/auto-merge.yml:4]: If not, let's get rid of probot

Comment (tmigot) [reply .github/auto-merge.yml:4]: Apparently not, so let's remove it. #92

**Pull request merged**